tlatools/org.lamport.tlatools/src/tla2sany/parser/Operators.java

```diff
@@ -148,4 +122,213 @@ public static void printTable() {
+    new Operator("<>",            40,  150, Operators.assocNone, Operators.prefix),
+    new Operator("ENABLED",       40,  150, Operators.assocNone, Operators.prefix),
+    new Operator("UNCHANGED",     40,  150, Operators.assocNone, Operators.prefix),
+    new Operator("SUBSET",        100, 130, Operators.assocNone, Operators.prefix),
+    new Operator("UNION",         100, 130, Operators.assocNone, Operators.prefix),
+    new Operator("DOMAIN",        100, 130, Operators.assocNone, Operators.prefix),
+    new Operator("^+",            150, 150, Operators.assocNone, Operators.postfix),
+    new Operator("^*",            150, 150, Operators.assocNone, Operators.postfix),
+    new Operator("^#",            150, 150, Operators.assocNone, Operators.postfix),
+    new Operator("\\cdot",        50,  140, Operators.assocLeft, Operators.infix),
+    new Operator("\\equiv",       20,  20,  Operators.assocNone, Operators.infix),
+    new Operator("-+->",          20,  20,  Operators.assocNone, Operators.infix),
+    new Operator("/=",            50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\subseteq",    50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\in",          50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\notin",       50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("<",             50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\leq",         50,  50,  Operators.assocNone, Operators.infix),
+    new Operator(">",             50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\geq",         50,  50,  Operators.assocNone, Operators.infix),
+    new Operator("\\times",       100, 130, Operators.assocLeft, Operators.nfix),
+    new Operator("\\",            80,  80,  Operators.assocNone, Operators.infix),
+    new Operator("\\intersect",   80,  80,  Operators.assocLeft, Operators.infix),
+    new Operator("\\union",       80,  80,  Operators.assocLeft, Operators.infix),
+    new Operator("...",           90,  90,  Operators.assocNone, Operators.infix),
+    new Operator("..",            90,  90,  Operators.assocNone, Operators.infix),
+    new Operator("|",             100, 110, Operators.assocLeft, Operators.infix),
+    new Operator("||",            100, 110, Operators.assocLeft, Operators.infix),
+    new Operator("&&",            130, 130, Operators.assocLeft, Operators.infix),
+    new Operator("&",             130, 130, Operators.assocLeft, Operators.infix),
+    new Operator("$$",            90,  130, Operators.assocLeft, Operators.infix),
+    new Operator("$",             90,  130, Operators.assocLeft, Operators.infix),
+    new Operator("??",            90,  130, Operators.assocLeft, Operators.infix),
+    // Removed as requested by Leslie (16 Feb. 01)
+    //new Operator("?",             90,  130, Operators.assocLeft, Operators.infix),
```

Comment:
This could be removed as well (IMO)

Reply:
I think it is interesting for historical reasons and @lemmy likes to keep comments of this sort around.